lib/plugins/physics.js

```diff
@@ -134,7 +136,8 @@ function inject (bot) {
     const inBlock = bot.blockAt(pos.offset(0, 0, 0))
     if (underBlock && underBlock.type === 88) {
       currentMaxGroundSpeed = physics.maxGroundSpeedSoulSand
-    } else if (inBlock && inBlock.type === 9) {
+    } else if (inBlock && (inBlock.type === 9 || inBlock.type === 11)) {
```

Comment:
Does it work the same for water and lava? isn't lava slower?

Reply:
lava isn't slower

Reply:
Could you retrieve the block IDs from minecraft data instead of hardcoding them ?